helm/templates/configmap.yaml

```diff
@@ -29,6 +29,9 @@ data:
   TILES_PATH: {{ .Values.env.tiles.path | quote }}
   TILES_UPLOAD_BATCH_SIZE: {{ .Values.env.tiles.uploadBatchSize | quote }}
   TILES_SIG_IS_NEEDED: {{ .Values.env.tiles.sigIsNeeded | quote }}
+  CRYPTO_READ_FILE_ENCODING: {{ .Values.env.crypto.readFileEncoding | quote }}
```

Comment:
you don't need the pem config?

Reply:
yes
they already put it in deployment (we don't want it configurable, path should be consistent)